tests/Integration/Framework/Test_ProxyManager.cfg

```diff
@@ -0,0 +1,123 @@
+Resources
+{
+  ProxyProviders
+  {
+    DIRAC_TEST_CA
+    {
+      ProxyProviderType = DIRACCA
+      CertFile = ../../../Core/Security/test/certs/ca/ca.cert.pem
```

Comment:
I assume here you are looking into all those files that you added in DIRAC/Core/Security/test. If those files are used only by integration tests please move them somewhere under DIRAC/tests/Integration.

What is in DIRAC/Core/Security/test should be unit test only (basically what is run in Travis) with pytest.

Reply:
Certs moved to Integration section